upstream-community-operators/banzaicloud-logging/banzaicloud-logging.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: banzaicloud-logging
+channels:
+  - name: beta
+    currentCSV: banzaicloud-logging.v0.1.8
```

Comment:
This should match the `metadata.name` of the CSV in this PR: `loggingoperator.v0.1.8`